scripts/d-i/late_command.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+
+set -eufx
+
+# This script setups ansible and runs it
+# It should be ran at the end of the basic installation of a machine
+
+apt install -y ansible git eatmydata
+
+# We clone our ansible repository and copy the ansible config files
+
+git clone https://github.com/CarlFK/video-stack-deploy.git /root/playbook-repo
```

Comment:
These are appropriate for you, but not us. You were planning to template this, right?